.github/workflows/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Cache node modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Install dependencies (npm ci)
+      run: npm ci
+      working-directory: ./phone-store
+    - name: Run unit tests (npm run test)
+      run: npm run test -- --no-watch --no-progress --browsers=ChromeHeadlessCI
+      working-directory: ./phone-store
+  e2e-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
The first four steps here are the same as on the `unit-tests` job. Is there a way to share those across the two jobs instead of repeating them?

Reply:
This is possible with [this](https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts) but it may not be worth doing as there is likely some overhead involved in packing up the artifacts and unpacking them in each of the test jobs.

Reply:
Yeah, that sounds too much like work for now. Let's skip it.